pymongo/helpers.py

```diff
@@ -221,8 +225,11 @@ def _check_write_command_response(result):
     if write_errors:
         _raise_last_write_error(write_errors)
 
-    error = result.get("writeConcernError")
+    error = result.get("writeConcernError", {})
```

Comment:
Minor nit: we can remove the `, {}` here because None is okay.

Reply:
Done. Changed this pattern elsewhere as well.